packages/contracts/contracts/test/Colosseum.t.sol

```diff
@@ -1141,7 +1141,7 @@ contract Colosseum_ValidatorSystemUpgrade_Test is Colosseum_Initializer {
     MockL2OutputOracle mockOracle;
     uint256 internal targetOutputIndex;
 
-    function setUp() public override {
+    function setUp() public virtual override {
```

Comment:
Why is the `virtual` added?

Reply:
This was made unnecessary by the change to the test contract. Updated.